x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/190_lookup_join.yml

```diff
@@ -12,8 +12,6 @@ setup:
       indices.create:
         index: test
         body:
-          settings:
-            number_of_shards: 5
```

Comment:
It's unrelated indeed. But this was "added" because I copied another file as a base.
It shouldn't be needed, as most tests don't have explicit shards, and I feel like it's better and cleaner without it.
Also, to avoid giving the impression of it being there "for a reason"